LearningMissionLab/Skill.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningMissionLab
+{
+    public class Skill
```

Comment:
You should consider creating an enum for skill names. If you want to have the level property you should create a separate enum for skill levels. Using ModuleLevel here is not desirable.

Reply:
Done.